book/website/communication/citable/citable-steps.md

```diff
@@ -0,0 +1,72 @@
+(rr-credit-make)=
+# Steps for Making Research Objects Citable
+
+There are many reasons why authors don't cite the data and software that they use, but one of the biggest ones is that it's not clear how.
+You can go a long way to reducing this barrier by following a few steps to make it as easy as possible.
+
+(rr-credit-make-object)=
+## 1. Identify Your Research Objects
+
+The first step is to ensure that you have something worth citing.
```

Comment:
Think this needs some wordsmithing as I expect plenty of people would say "oh my dataset/code isn't worth making citable" and we want to avoid that!